### DISCUSSION TITLE
Daily 24-03-2022

### DISCUSSION THREAD

**Discussion by YaakovHatam** [YaakovHatam/diet-recipes#16] [Show and tell]
Daily

**Comment by Amicht** [comment]
I have finished the 'search screen', its props are - 
1. the Database of all the manus,
2. the state function that defines the user`s custom-menus (should be created from the app function).

**Comment by ShlomoRussell** [comment]
I finished my functions unless I misunderstood what I was supposed to do. I am just waiting to see where I am pulling the info to the functions from. But other than that I think I am finished.
